pxc-80-backup/Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM registry.access.redhat.com/ubi7/ubi-minimal
+
+LABEL org.opencontainers.image.release="2.4"
+LABEL org.opencontainers.image.title="Percona XtraBackup"
+LABEL org.opencontainers.image.vendor="Percona"
+LABEL org.opencontainers.image.description="Percona XtraBackup works with MySQL, MariaDB, and \
+Percona Server. It supports completely non-blocking backups of InnoDB, XtraDB, and HailDB storage engines. \
+In addition, it can back up the following storage engines by briefly pausing writes at the end of the backup:  \ MyISAM, Merge, and Archive, including partitioned tables, triggers, and database options."
+LABEL org.opencontainers.image.license="Apache 2.0"
+LABEL org.opencontainers.image.authors="info@percona.com"
+LABEL org.opencontainers.image.maintainer="Percona Development <info@percona.com>"
+
+# check repository package signature in secure way
+RUN export GNUPGHOME="$(mktemp -d)" \
+    && gpg --batch --keyserver pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A \
+    && gpg --batch --keyserver pool.sks-keyservers.net --recv-keys 6341AB2753D78A78A7C27BB124C6A8A7F4A80EB5 \
+    && gpg --batch --keyserver pool.sks-keyservers.net --recv-keys 91E97D7C4A5E96F17F3E888F6A2FAEA2352C64E5 \
+	&& gpg --export --armor 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A > ${GNUPGHOME}/RPM-GPG-KEY-Percona \
+	&& gpg --export --armor 6341AB2753D78A78A7C27BB124C6A8A7F4A80EB5 > ${GNUPGHOME}/RPM-GPG-KEY-CentOS-7 \
+	&& gpg --export --armor 91E97D7C4A5E96F17F3E888F6A2FAEA2352C64E5 > ${GNUPGHOME}/RPM-GPG-KEY-EPEL-7 \
+	&& rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona ${GNUPGHOME}/RPM-GPG-KEY-CentOS-7 ${GNUPGHOME}/RPM-GPG-KEY-EPEL-7 \
+	&& curl -Lf -o /tmp/percona-release.rpm https://repo.percona.com/yum/percona-release-latest.noarch.rpm \
+	&& rpmkeys --checksig /tmp/percona-release.rpm \
+	&& rpm -i /tmp/percona-release.rpm \
+	&& rm -rf "$GNUPGHOME" /tmp/percona-release.rpm \
+	&& rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY
+
+# install exact version of PS for repeatability
+ENV XTRABACKUP_VERSION 8.0.9-1.el7
+ENV PS_VERSION 8.0.18-9.1.el7
+ENV PXC_VERSION 8.0.18-26.4.3.1.el7
```

Comment:
new version is 8.0.18-26.4.3.2.

Reply:
https://github.com/percona/percona-docker/pull/158/commits/de507c4f5764f16946f4237e3c92bde1ee0c917c